integrated_channels/api/v1/logs/serializers.py

```diff
@@ -56,6 +58,14 @@ def get_sync_last_attempted_at(self, obj):
         else:
             return max(res)
 
+    def get_friendly_status_message(self, obj):
+        """
+        Return a human readable status string.
+        """
+        if obj.api_response_status_code is not None and HTTP_STATUS_STRINGS.get(obj.api_response_status_code):
+            return HTTP_STATUS_STRINGS[obj.api_response_status_code]
```

Comment:
I would probably use the getter here instead of direct key fetching just in case we get a weird HTTP status code